modules/ossm-cert-manager-update-istio-csr-revisionbased-only.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="updating-istio-csr-revision-based-only_{context}"]
+= Updating istio-csr for RevisionBased update strategy
+
+When you created your `Istio` resource, if you set the `spec.updateStrategy` field to `RevisionBased`, the `istio-csr` agent requires that you pass all revisions each time you upgrade your `RevisionBased` control plane. To do so, you must follow this order each time you upgrade:
+
+. Update the `istio-csr` deployment with new revision.
+. Update the `Istio.spec.version` value.
+
+[id="example-update-revision-based-control-plane_{context}"]
+== Example update for RevisionBased control plane
```

Comment:
This isn't formatted as a proper procedure module. And this really doesn't need a second heading. 

You could refactor the commands using variables and a combination of code callouts and a definition list.